### ISSUE TITLE
--comments not working

### ISSUE THREAD

**Issue by lucybeebee** [nlitsme/youtube_tool#11]
e.g.  yttool --comments bJOuzqu3MUQ

Output is empty. Please fix. thank you very much!

**Comment by terraformr** [comment]
Yeah, I'm having this same issue as lucybeebee. Looks like you had this problem historically as well but managed to fix it. Hopefully you can again!

Basically, example query:

yttool --comments https://www.youtube.com/watch?v=testurl > testurl.txt

is outputting nothing but a blank .txt file

**Comment by Plaidstallion** [comment]
same issue
